File: Count of Smaller Numbers After Self/CoSNAS.py
def countSmaller(nums):
    rlist = []
    while(nums[0]):
        try:
            nums[1]
        except IndexError:
            time =0
            rlist.append(time)
            break
        else:
            time = 0
            a = nums[0]
            print(a)
            nums = nums[1:]
            print(nums)
            for i in nums:
                if i <a:
                    time +=1
            rlist.append(time)

    return rlist

File: Count-of-Smaller-Numbers-After-Self/CoSNAS.go
package main

import (
	"fmt"
)

func countSmaller(nums []int) []int {
	var nums2 []int

	for i, numb := range nums {
		timeCount := 0
		for ii := i; ii < len(nums); ii++ {
			if nums[ii] < numb {
				timeCount = timeCount + 1
			}
		}
		nums2 = append(nums2, timeCount)
	}

	return nums2

}

func main() {
	args := []int{5, 2, 6, 1}
	fmt.Println(countSmaller(args))
}
